src/interfaces/SpokePool.ts

```diff
@@ -47,8 +47,8 @@ export interface v3DepositWithBlock extends v3Deposit, SortableEvent {
   quoteBlockNumber: number;
 }
 
-export type Deposit = v2Deposit; // @todo: Extend with v2Deposit | v3Deposit.
-export type DepositWithBlock = v2DepositWithBlock; // @todo Extend with v2DepositWithBlock | v3DepositWithBlock.
+export type Deposit = v2Deposit | v3Deposit;
+export type DepositWithBlock = v2DepositWithBlock | v3DepositWithBlock;
```

Comment:
Brought up offline but wdyt about moving `realizedLpFeePct` into the v2Deposit type and out of DepositCommon since we won't be able to compute it upfront until we match a fill with a deposit once we implement https://linear.app/uma/issue/ACX-1838/update-lp-fee-computation-to-replace-destinationchainid-with

Reply:
We'll tackle this in a follow up commit :+1: